### ISSUE TITLE
Fails silently

### ISSUE THREAD

**Issue by valeriob** [madskristensen/TypeScriptDefinitionGenerator#19]
### Installed product versions
- Visual Studio: [example 2017 Enterprise]
- This extension: [example 1.0.12]

### Description
Upon .cs file save, the extension files silently, the file is not generated and i can't understand why.

### Steps to recreate
I do not know what am i doing wrong since i have no error output, but just try to generate something like a class with subclass and it will manifest itself.

### Current behavior
It fails silently

### Expected behavior
At least report what pattern is not supported that causes the failure....

**Comment by raboud** [comment]
Can you supply the cs file that makes it fail? I have a fix for the Embodied failure.  issue #9 (not sure why it got closed. 

I would like to add logging to the Errors window...I am investigating how to make that happen.

**Comment by valeriob** [comment]
I don't think i can, but if you have a version that output something i'm willing to try it out :D

**Comment by raboud** [comment]
do the classes have properties that uses Expression-bodied Members.

```
public class GridWebClient
{
    public Uri Url => new Uri($"/clients/foo", UriKind.Relative);
}
```
the => means they are Expression-bodied members

there is an issue with VS and theses....I have a work around for TypeScriptDefinitionGenerator and a proper fix going into Roslyn 15.3

**Comment by valeriob** [comment]
Nope i do not have any of that

**Comment by raboud** [comment]
well without a sample of a class that fails....its hard to fix

**Comment by valeriob** [comment]
You are right, but it's quite a large file with many classes, it will take some time to diagnose.
Well all would be much much easier if the tool would fail with some error message, see #19

**Comment by valeriob** [comment]
I'm trying to diagnose it, but every time a save a file it freezes vs 2017 for more than 1 minute...
I'll send you the file, maybe you can find out the problem very quickly

**Comment by raboud** [comment]
Please Try Build 1.1.24 to see if your issues are corrected.
